app/scripts/app-init.js

```diff
@@ -25,14 +23,16 @@ function importAllScripts() {
     tryImport('./lockdown-more.js');
     tryImport('./policy-load.js');
   } else {
+    tryImport('./init-globals.js');
     tryImport('./lockdown-install.js');
-    tryImport('./lockdown-more.js');
     tryImport('./lockdown-run.js');
+    tryImport('./lockdown-more.js');
     tryImport('./runtime-cjs.js');
   }
 
   const fileList = [
     // The list of files is injected at build time by replacing comment below with comma separated strings of file names
+    // https://github.com/MetaMask/metamask-extension/blob/develop/development/build/scripts.js#L406
```

Comment:
if you're going to put a reference on github, press the y key so you can lock to a commit and the line number is fixed to what you want to show

Reply:
so useful I did not knew this before